clang/unittests/Format/FormatTest.cpp

```diff
@@ -10093,11 +10090,11 @@ TEST_F(FormatTest, ReturnTypeBreakingStyle) {
                getGoogleStyleWithColumns(40));
   verifyFormat("Tttttttttttttttttttttttt ppppppppppppppp\n"
                "    ABSL_GUARDED_BY(mutex1)\n"
-               "    ABSL_GUARDED_BY(mutex2);",
+               "        ABSL_GUARDED_BY(mutex2);",
```

Comment:
This looks wrong.

Reply:
It is wrong, but it was like the recent patch where I changed Google style. I mentioned it in the commit description  (quote follows):

> Also update the test with two GUARDED_BY directives. While the formatting after https://github.com/llvm/llvm-project/commit/efeb546865c233dfa7706ee0316c676de9f69897 seems better, this case is rare enough to not warrant the extra complexity. We are reverting it back to the state it had before
https://github.com/llvm/llvm-project/commit/efeb546865c233dfa7706ee0316c676de9f69897.